### PR TITLE
Updated Button Control

### PR DESCRIPTION
**Pull request by OpResCodes**
I have updated the dependencyproperties of the button. Also I changed the event subscription to the static timer to prevent potential memory leaks. #2

### CODE REVIEW

**Comment by OpResCodes** [comment]
Ok, I also updated the rest of the controls. :)

File: LCARSToolkit/LCARSToolkit.Controls/Button.cs [hunk 1]
```diff
@@ -19,6 +19,28 @@ namespace LCARSToolkit.Controls
...
+        }
+
+        private void FlashTimer_Tick(object sender, object e)
+        {
+            IsLit = (Illumination == Illumination.Flashing) ? !IsLit : (Illumination == Illumination.On);
+        }
+
+        private void Button_Unloaded(object sender, RoutedEventArgs e)
+        {
+            Extensions.FlashTimer.Tick -= FlashTimer_Tick;
```
Comment (michaelosthege) [inline LCARSToolkit/LCARSToolkit.Controls/Button.cs:41]: Is it necessary to detach the `Button_Unloaded` handler too?

**Review by michaelosthege — APPROVED** [review]

File: LCARSToolkit/LCARSToolkit.Controls/Button.cs (hunk 1 above)
Comment (OpResCodes) [reply LCARSToolkit/LCARSToolkit.Controls/Button.cs:41]: It should be enough to cleanup references to this object on other objects (in this case the reference to the button on the static flashtimer). Once no external references reach the object it gets garbage collected.

**Comment by michaelosthege** [comment]
Awesome, thanks for the PR @OpResCodes !

**Comment by michaelosthege** [comment]
@OpResCodes I just updated the NuGet package. please let me know if it works as expected

**Comment by OpResCodes** [comment]
Thanks, all seems to work fine now!

**Pull request merged**